entrypoint.sh

```diff
@@ -75,9 +78,10 @@ _add_files() {
     echo "::debug::Apply add options ${INPUT_ADD_OPTIONS}";
 
     echo "INPUT_FILE_PATTERN: ${INPUT_FILE_PATTERN}";
+    INPUT_FILE_PATTERN_ARRAY=( ${INPUT_FILE_PATTERN} )
 
     # shellcheck disable=SC2086
-    git add ${INPUT_ADD_OPTIONS} ${INPUT_FILE_PATTERN};
+    git add ${INPUT_ADD_OPTIONS} ${INPUT_FILE_PATTERN:+"${INPUT_FILE_PATTERN_ARRAY[@]}"};
```

Comment:
```suggestion
    git add ${INPUT_ADD_OPTIONS} "${INPUT_FILE_PATTERN_ARRAY[@]}";
```

No that the case of INPUT_FILE_PATTERN_ARRAY being empty seems well handled already by the "${array[@]}" notation.